conf/application.conf

```diff
@@ -145,7 +145,7 @@ microservice {
     features {
       isGuaranteeBalanceEnabled = true
       phase5Enabled {
-        departure = false
+        departure = true
```

Comment:
Need to change this back

Reply:
Yeah that'll need to be set back to false